### DISCUSSION TITLE
No content in response when creating a new user

### DISCUSSION THREAD

**Discussion by jmjoary** [directus/directus#15096] [Archive]
### Describe the Bug

In the api doc, we can see in the [doc](https://docs.directus.io/reference/system/users.html#create-a-user) that the user creation process must answer with the user object. When I do it, I receive the HTTP code 204 because I don't get any content. But the user is created.

### To Reproduce

Send a Post request to your server with this informations :

- path : /users
- body : {"email":"test@test.com","password":"password"}

### Errors Shown

Here is the response I got :

> HTTP/1.1 204 No Content
Server: nginx/1.18.0 (Ubuntu)
Date: Tue, 16 Aug 2022 16:15:48 GMT
Connection: close
Content-Security-Policy: script-src 'self' 'unsafe-eval';worker-src 'self' blob:;child-src 'self' blob:;img-src 'self' data: blob: https://cdn.directus.io;media-src 'self' https://cdn.directus.io;connect-src 'self' https://*;default-src 'self';base-uri 'self';block-all-mixed-content;font-src 'self' https: data:;frame-ancestors 'self';object-src 'none';script-src-attr 'none';style-src 'self' https: 'unsafe-inline'
X-Powered-By: Directus
Vary: Origin, Cache-Control
Access-Control-Allow-Credentials: true
Access-Control-Expose-Headers: Content-Range
Cache-Control: no-cache

### What version of Directus are you using?

9.15.1

### What version of Node.js are you using?

16.15.1

### What database are you using?

MariaDB 10.3.34-MariaDB-0ubuntu0.20.04.1

### What browser are you using?

api

### How are you deploying Directus?

locally

**Comment by rijkvanzanten** [comment]
This is most likely because the user you're using to create the user doesn't have read access to users 🙂

**Reply by jmjoary** [reply]
My bad. Noob question. Thanks for the answer.
